_data/tagging-status.yml

```diff
@@ -2753,13 +2753,13 @@
 
  - name: eepic
    type: package
-   status: unknown
+   status: compatible
    included-in: [tlc3, arxiv01]
    priority: 2
    issues:
-   tests: false
-   tasks: needs tests
-   updated: 2024-07-14
+   tests: true
+   comments: "Does not work with pdflatex or lualatex."
```

Comment:
I thought it would work with pdflatex

Reply:
For me the eepic-specific commands like `\ellipse` don't show in the pdf with pdflatex

Reply:
yes that's true. according to the pdftex manual Hans Hagen had some macros (probably similar to the ones graphicx still uses for metapost) to directly interpret tpic specials, which would allow this to work but by now it's probably better to let this fall into obscurity rather than dig out the old pdftex support.